config/services.yaml

```diff
@@ -127,19 +127,19 @@ services:
 
     Github\HttpClient\Builder:
         arguments:
-            - '@?Http\Client\HttpClient'
-            - '@?Http\Message\RequestFactory'
-            - '@?Http\Message\StreamFactory'
+            - '@Psr\Http\Client\ClientInterface'
+            - '@Psr\Http\Message\RequestFactoryInterface'
+            - '@Psr\Http\Message\StreamFactoryInterface'
```

Comment:
I assume all of the optional-to-non-optional changes are because the services are now always registered?

Reply:
Yes they are indeed, I found at that the wrong service was getting injected, so I figured it would be best to just specifically point to the interfaces instead